.github/workflows/tests.yml

```
@@ -2,9 +2,7 @@ name: Tests
 
 on:
   push:
-    branches: [main week02]
-  pull_request:
-    branches: [main]
+    branches: [week03]
```

Change `on` to this and it your tests should run on every branch when a push happens, which is what you want.
```suggestion
on: [push]
```